src/Sylius/Bundle/ApiBundle/Resources/config/app/config.yaml

```diff
@@ -52,3 +52,9 @@ lexik_jwt_authentication:
             enabled: true
             prefix: Bearer
             name: "%sylius.api.authorization_header%"
+
+sylius_mailer:
+    emails:
+        contact_request_api:
+            subject: sylius.emails.contact_request.subject
+            template: "@SyliusApi/Email/contactRequest.html.twig"
```

Comment:
Is this email used?

Reply:
https://github.com/Sylius/Sylius/pull/13960#discussion_r883337617